streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -21,17 +21,22 @@ import java.io.{File, NotSerializableException}
 import java.util.concurrent.atomic.AtomicInteger
 
 import org.apache.commons.io.FileUtils
+import org.apache.spark.metrics.MetricsSystem
+import org.apache.spark.metrics.source.Source
```

Comment:
These need to be with other spark imports, and the spark imports shouldn't have a blank line in the middle.